app.js

```diff
@@ -13,14 +13,18 @@ var apos = require('apostrophe')({
 
     // Note: most configuration occurs in the respective
     // modules' directories. See lib/apostrophe-assets/index.js for an example.
-    
+
     // However any modules that are not present by default in Apostrophe must at
     // least have a minimal configuration here: `moduleName: {}`
 
     // If a template is not found somewhere else, serve it from the top-level
     // `views/` folder of the project
+    'apostrophe-templates': { viewsFolderFallback: path.join(__dirname, 'views') },
 
-    'apostrophe-templates': { viewsFolderFallback: path.join(__dirname, 'views') }
-
+    // Add an alt field to images, by default the title is used but
+    // we recommend using the alt one with this options for more clarity.
```

Comment:
"we recommend enabling the alt field for clarity."

Reply:
Done